src/presenters/abandon-radio.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function (translate) {
+  return {
+    id: "prove-identity-another-way",
+    name: "abandonRadio",
+    label: translate("pages.prove-identity-another-way.legend"),
+    legend: translate("pages.prove-identity-another-way.legend"),
+    fieldset: {
+      legend: {
+        isPageHeading: true,
+        classes: "govuk-fieldset__legend--m",
+      },
+    },
+    hint: {
+      html: " ",
```

Comment:
Does this need to be provided? I feel like this could be an accessibility issue, say if a screenreader tries to read an empty hint